tests/audio.rs

```diff
@@ -0,0 +1,50 @@
+use syrillian::audio::audio:: {
+    AudioScene, AudioSystem
+};
+
+#[test]
+fn test_load_sounds() {
+    let mut audio = AudioSystem::new();
+    audio.load_sound("pop", "examples/assets/pop.wav");
```

Comment:
As stated earlier, i do think the Sounds belong in to their own Asset Store.

But maybe- we can have both. I do like the referencing of assets by some simple name, and i can see users making a mapping like that themselves. So why not already provide it in the audio system.

But what i'm saying is, the AudioSystem can load the data, and then it'll simply use an Asset Store as its backing storage.